IFC4x3/Properties/m/MassPerLength_3veHedEdf31BnVpNC7t3EA/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MassPerLength_3veHedEdf31BnVpNC7t3EA" Name="MassPerLength" UniqueId="f9a11a27-3a7a-4304-bc5f-cd7307dc338a" PrimaryDataType="IfcMassPerLengthMeasure" />
```

Comment:
Reuse?

Reply:
So firstly reuse, `MassPerLength_1bxC00qVOHuO00025QrEV`, and I would highly suggest considering, PSet_ProfileMechanical when it comes to mechanical properties on segments and profile objects